conform-pr/main.go

```diff
@@ -87,9 +93,33 @@ type checkResult struct {
 	err   error
 }
 
-// runChecks runs all the checks on the given PR GraphQL node.
-func runChecks(ctx context.Context, action *githubactions.Action, client *graphql.Client, nodeID string) []checkResult {
-	pr := client.GetPullRequest(ctx, nodeID)
+// runChecks runs all the checks for the given PR.
+//
+//nolint:lll // package and type names are long
+func runChecks(ctx context.Context, action *githubactions.Action, client *github.Client, gClient *graphql.Client, org, user, nodeID string) ([]checkResult, bool) {
```

Comment:
I'm a bit skeptical about the new signature - that second result parameter doesn't look super relevant to what the function is doing. I understand the idea - as we pass org and user to the function, we can identify if the user is a part of the org, and some checks might depend on it, but still it's a little bit outside of the service of this function. I don't have a better idea though, so I don't make this a change request.

Reply:
Another aspect of that – it makes testing easier